README.md

```diff
@@ -274,91 +274,91 @@ const executor = async (resolve, reject, resolveWithRemote) => {
+const targetP = Promise.delegate(executor, unfulfilledHandler);
 E(E(targetP).remoteMethod(someArg, someArg2)).callOnResult(...otherArgs);
 ```
 
-The handlers are not exposed to the user of the handled promise, so it provides
+The handlers are not exposed to the user of the delegated promise, so it provides
 a secure separation between the unprivileged client (which uses the `E`,
-`E.sendOnly` or static `HandledPromise` methods) and the privileged system
+`E.sendOnly` or static `Promise` methods) and the privileged system
 which implements the communication mechanism.
 
-### `HandledPromise.prototype`
+### `Promise.prototype`
 
-Although `HandledPromise` is class-like, it is not intended to act like a class
-distinct from `Promise`.  The initial value of `HandledPromise.prototype` is
-the same as the initial value of `Promise.prototype`.  The initial value of `HandledPromise.prototype.constructor` remains `Promise`.  Code that holds a promise cannot sense whether it holds a handled or an unhandled promise.
+Since delegated promises are still Promises, they can be used anyplace a
+Promise can.  However, with the additional semantics of `Promise.resolve`,
+it is possible to detect if an object is a presence.
 
 
 ### Handler traps
 
-A handler object can provide handler traps (`get`, `applyFunction`,
-`applyMethod`) and their associated `*SendOnly` traps.
+A handler object can provide handler traps (`eventualGet`, `eventualApply`,
+`eventualSend`) and their associated `*Only` traps.
 
 ```ts
 ({
-  get                   (target, prop):        Promise<result>,
-  getSendOnly           (target, prop):        void,
-  applyFunction         (target, args):        Promise<result>,
-  applyFunctionSendOnly (target, args):        void,
-  applyMethod           (target, prop, args):  Promise<result>,
-  applyMethodSendOnly   (target, prop, args):  void,
+  eventualGet        (target, prop):        Promise<result>,
+  eventualGetOnly    (target, prop):        void,
+  eventualApply      (target, args):        Promise<result>,
+  eventualApplyOnly  (target, args):        void,
+  eventualSend       (target, prop, args):  Promise<result>,
+  eventualSendOnly   (target, prop, args):  void,
 })
 ```
 
-If the handler does not provide a `*SendOnly` trap, its default implementation
+If the handler does not provide a `*Only` trap, its default implementation
 is the non-send-only trap with a return value of `undefined` (not a promise).
 
-If the handler omits a non-send-only trap, invoking the associated operation
+If the handler omits a non-only trap, invoking the associated operation
 returns a promise rejection.  The only exception to that behaviour is if the
-handler does not provide the `applyMethod` optimization trap.  Then, its
+handler does not provide the `eventualApply` optimization trap.  Then, its
 default implementation is
 ```js
-HandledPromise.applyFunction(HandledPromise.get(p, prop), args)
+Promise.applyFunction(Promise.get(p, prop), args)
```

Comment:
I think this needs to be changed from `Promise.get` to `Promise.eventualGet`?

Reply:
Thanks, there were a few other things wrong with this section, too.